### DISCUSSION TITLE
"Open Drive"?

### DISCUSSION THREAD

**Discussion by MattiasMartens** [HexFiend/HexFiend#498] [Q&A]
I see some mentions in GitHub Issues of an "Open Drive" feature but no documentation for the feature itself. When I open the latest version of HexFiend on Mac OS Sequoia I don't see any option to open a drive, only individual files which is not what I am looking for.

Does this feature exist? Does it depend on a certain OS version to be active?

**Comment by kainjow** [comment]
Are you using the App Store version? if so the feature doesn’t exist there. You need to use the GitHub version.
